flyerhq/flutter_chat_ui
# [Feature] directionality and localization

### What does it do?

Implement directionality where if the text direction is RTL everything moves according to it.
I also added Arabic Localization as `ChatL10nAr`

### Why is it needed?

The chat to be useable in RTL languages the bubbles and buttons need to be swapped by the Y-axis

### How to test it?

Wrap the `Chat` widget with `Directionality` and change TextDirection from ltr to rtl to see the effect.
ex:
```dart
Directionality(
    textDirection: TextDirection.rtl,
    child: Chat()
)
```
